src/common/check_dataset_schema/test.py

```diff
@@ -3,12 +3,54 @@
 import json
 
 input_path = meta["resources_dir"] + "resources_test/common/pancreas/dataset.h5ad"
-input_correct_schema = meta["resources_dir"] +  "resources_test/common/check_schema/anndata_correct.yaml"
-input_error_schema = meta["resources_dir"] + "resources_test/common/check_schema/anndata_error.yaml"
+input_correct_schema = meta["resources_dir"] +  "resources_test/common/anndata_correct.yaml"
+input_error_schema = meta["resources_dir"] + "resources_test/common/anndata_error.yaml"
```

Comment:
No need to use the `meta["resources_dir"]`

```suggestion
input_correct_schema = "anndata_correct.yaml"
input_error_schema = "anndata_error.yaml"
```